example/src/App.js

```diff
@@ -3,15 +3,19 @@ import React, { Component } from "react";
 import ICalLink from "react-icalendar-link";
 
 export default class App extends Component {
-  render() {
+  render () {
```

Comment:
Do you use prettier? I notice the formatting below is a bit different than I expect. Can you help me to make sure this? 
![image](https://user-images.githubusercontent.com/136648/118978392-9e8c9100-b9ba-11eb-86fd-04fd57fc7737.png)